WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -59,6 +83,21 @@ protected override async Task ActionAsync(CancellationToken cancel)
 
 				// Ensure there's at least one non-blame round in input registration.
 				await CreateRoundsAsync(cancel).ConfigureAwait(false);
+
+				foreach (var round in RoundsById.Select(x => x.Value))
+				{
+					if (round.Phase != Phase.Ended)
```

Comment:
Sure, but this is a horrible hack that must be removed ASAP, along with the concurrent dictionary values view, the purpose of this is so that when a test injects an `Alice` into a round in the middle of some process, that that `Alice` is considered fully registered, an implementation detail assumption that comes up repeatedly. The sooner we can delete this the better and I don't think it's worth spending time making it nicer in general (although this particular suggestion is trivial).